docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+version: "3"
+
+services:
+  catalog-view-api:
+    image: eu.gcr.io/digdir-fdk-infra/catalog-view-api:test
+    build: .
+    environment:
+      - MONGO_USERNAME=admin
```

Comment:
Har disse hardkodete-verdiene ingen effekt i prod-miljet?

Reply:
docker-compose er kun brukt for å kjøre opp lokalt